CRM/Banking/Page/Payments.php

```diff
@@ -265,7 +265,7 @@ public function build_statementPage($payment_states) {
         'sequence' => $stmt->sequence,
         'total' => $stmt->total,
         'currency' => $stmt->currency,
-        'date' => strtotime($stmt->starting_date),
+        'date' => strtotime($stmt->starting_date ?? '2000-01-01 00:00:00'),
```

Comment:
What's the reason for the chosen fallback? Could it also be `0000-00-00` or `01-01-1970`?